src/applications/vaos/tests/e2e/community-care.cypress.spec.js

```diff
@@ -165,9 +165,17 @@ describe('Create new community care appointment', () => {
       let date = moment().add(5, 'days');
 
       if (date.weekday() === 0) {
-        date = moment().add(7, 'days');
+        date = date
+          .add(7, 'days')
+          .day(1)
```

Comment:
Would doing `.add(1, 'days')` work here instead of these two lines? Since we know we just need to pick Monday, which is the next day.

Reply:
I suppose it would :P